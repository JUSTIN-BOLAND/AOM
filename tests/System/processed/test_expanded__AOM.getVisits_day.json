[{"visitorId":"8834336047278127061","visitId":"2","firstActionTime":"2015-12-01 01:29:45","device":"desktop","source":"direct","refererName":"","refererKeyword":"","refererUrl":"","adData":{"platform":"Criteo","campaignId":"4711"}},{"visitorId":"3479285623182512786","visitId":"4","firstActionTime":"2015-12-01 01:29:45","device":"desktop","source":"direct","refererName":"","refererKeyword":"","refererUrl":"","adData":{"platform":"AdWords","campaignId":"184418636","adGroupId":"9794351276","targetId":"kwd-118607649","creative":"47609133356","placement":"","network":"g","device":"m","adPosition":"1t2","locPhysical":"20228","locInterest":"1004074"}},{"visitorId":"4487853443349803806","visitId":"6","firstActionTime":"2015-12-01 01:29:45","device":"desktop","source":"direct","refererName":"","refererKeyword":"","refererUrl":"","adData":{"platform":"Criteo","campaignId":"14340"}},{"visitorId":"5083436001463556729","visitId":"1","firstActionTime":"2015-12-01 01:29:45","device":"desktop","source":"direct","refererName":"","refererKeyword":"","refererUrl":"","adData":{"platform":"Criteo","campaignId":"14340"}},{"visitorId":"5464597735718542573","visitId":"7","firstActionTime":"2015-12-01 01:35:45","device":"desktop","source":"direct","refererName":"","refererKeyword":"","refererUrl":"","adData":{"platform":"FacebookAds","campaignGroupId":"4160286035775","campaignId":"6028603577541","adGroupId":"5760286037541"}},{"visitorId":"8834336047278127061","visitId":"3","firstActionTime":"2015-12-01 01:41:45","device":"desktop","source":"direct","refererName":"","refererKeyword":"","refererUrl":"","adData":{"platform":"Criteo","campaignId":"84571"}},{"visitorId":"13291187357687896989","visitId":"5","firstActionTime":"2015-12-01 01:41:45","device":"desktop","source":"direct","refererName":"","refererKeyword":"","refererUrl":"","adData":{"platform":"Bing","campaignId":"190561279","adGroupId":"2029114499","orderItemId":"40414589411","targetId":"40414589411","adId":"5222037942"}}]