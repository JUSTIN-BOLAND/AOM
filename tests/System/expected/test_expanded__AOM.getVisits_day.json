[{"visitorId":"8834336047278127061","visitId":"2","firstActionTime":"2015-12-01 01:29:45","device":"desktop","source":"direct","refererName":null,"refererKeyword":null,"refererUrl":"","adParams":{"platform":"Criteo","campaignId":"4711"},"adData":[]},{"visitorId":"3479285623182512786","visitId":"4","firstActionTime":"2015-12-01 01:29:45","device":"desktop","source":"direct","refererName":null,"refererKeyword":null,"refererUrl":"","adParams":{"platform":"AdWords","campaignId":"184418636","adGroupId":"9794351276","feedItemId":"","targetId":"kwd-118607649","creative":"47609133356","placement":"","target":"","network":"g","adPosition":"1t2","locPhysical":"20228","locInterest":"1004074"},"adData":[]},{"visitorId":"4487853443349803806","visitId":"6","firstActionTime":"2015-12-01 01:29:45","device":"desktop","source":"direct","refererName":null,"refererKeyword":null,"refererUrl":"","adParams":{"platform":"Criteo","campaignId":"14340"},"adData":[]},{"visitorId":"5083436001463556729","visitId":"1","firstActionTime":"2015-12-01 01:29:45","device":"desktop","source":"direct","refererName":null,"refererKeyword":null,"refererUrl":"","adParams":{"platform":"Criteo","campaignId":"14340"},"adData":[]},{"visitorId":"5464597735718542573","visitId":"7","firstActionTime":"2015-12-01 01:35:45","device":"desktop","source":"direct","refererName":null,"refererKeyword":null,"refererUrl":"","adParams":{"platform":"FacebookAds","campaignId":"4160286035775","adsetId":"6028603577541","adId":"5760286037541"},"adData":[]},{"visitorId":"8834336047278127061","visitId":"3","firstActionTime":"2015-12-01 01:41:45","device":"desktop","source":"direct","refererName":null,"refererKeyword":null,"refererUrl":"","adParams":{"platform":"Criteo","campaignId":"84571"},"adData":[]},{"visitorId":"13291187357687896989","visitId":"5","firstActionTime":"2015-12-01 01:41:45","device":"desktop","source":"direct","refererName":null,"refererKeyword":null,"refererUrl":"","adParams":{"platform":"Bing","campaignId":"190561279","adGroupId":"2029114499","orderItemId":"40414589411","targetId":"40414589411","adId":"5222037942"},"adData":[]}]